Resources/Prototypes/Entities/Objects/Specific/chemical-containers.yml

```
@@ -83,6 +83,60 @@
       tags:
       - ChemDispensable
     - type: DnaSubstanceTrace
+- type: entity
+  name: tanque de reagente
+  parent: BaseItem
+  description: Um tanque metálico feito para segurar uma quantidade enorme de soluções, há um espaço onde uma seringa pode ser inserida.
+  id: TanqueDeReagenteMelhor
+  components:
+  - type: RefillableSolution
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)